src/statusSelectionCount.ts

```diff
@@ -46,11 +46,11 @@ export default class StatusSelectionCount extends Disposable {
 		const nFocus = focused !== undefined ? numberFormat.format(focused) : undefined;
 		const nSelected = selected > 1 ? numberFormat.format(selected) : undefined;
 		if (nFocus && nSelected) {
-			this.item.text = vscode.l10n.t("Byte {0} ({1} selected)", nFocus, nSelected);
+			this.item.text = vscode.l10n.t("Byte 0d{0}/0x{1} (0d{2}/0x{3} selected)", nFocus, nFocus.toString(16).toUpperCase(), nSelected, nSelected.toString(16).toUpperCase());
 		} else if (nSelected) {
-			this.item.text = vscode.l10n.t("{0} selected", nSelected);
+			this.item.text = vscode.l10n.t("0d{0}/0x{1} selected", nSelected, nSelected.toString(16).toUpperCase());
 		} else if (nFocus) {
-			this.item.text = vscode.l10n.t("Byte {0}", nFocus);
+			this.item.text = vscode.l10n.t("Byte 0d{0}/0x{1}", nFocus, nFocus.toString(16).toUpperCase());
```

Comment:
The `0d` prefix is uncommon, we can probably leave it out -- people will assume base 10 if there's no prefix

Reply:
Yep, you're right